build.sbt

```diff
@@ -251,3 +252,25 @@ lazy val parboiledCore = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     Compile / sourceGenerators += generateActionOps.taskValue
   )
   .nativeSettings(nativeSettings)
+
+ThisBuild / githubWorkflowTargetTags ++= Seq("v*")
+ThisBuild / githubWorkflowPublishTargetBranches := Seq()
```

Comment:
This disables publishing for now, if you want to use sbt-github-actions for publishing just let me know and I will set it up in another PR.